src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
 public class SalaryInfo {
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int HOURS_WORKED_INDEX = 2;
+    private static final int HOURLY_RATE_INDEX = 3;
+    private static final int NUMBER_OF_MONTH = 0;
+    private static final int MONTH = 1;
+    private static final int YEAR = 2;
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder resultBuilder = new StringBuilder();
+        resultBuilder.append("Report for period ").append(dateFrom).append(" - ").append(dateTo).append(System.lineSeparator());
+        String [] arrayDateFrom = dateFrom.split("\\.");
+        String [] arrayDateTo = dateTo.split("\\.");
+        LocalDate localDateFrom = LocalDate.of(Integer.parseInt(arrayDateFrom[YEAR]),
+                Integer.parseInt(arrayDateFrom[MONTH]), Integer.parseInt(arrayDateFrom[NUMBER_OF_MONTH]));
+        LocalDate localDateTo = LocalDate.of(Integer.parseInt(arrayDateTo[YEAR]),
```

Comment:
What do you think about using java [formatter](https://docs.oracle.com/javase/tutorial/datetime/iso/format.html) for these purposes? I would say, it definitely improves readability, flexibility and consistency of the code.